website_snippet_preset/static/src/js/website_snippet_preset.js

```diff
@@ -1,44 +1,112 @@
 /* Copyright 2018 Onestein
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
 
-odoo.define('website_snippet_preset', function(require) {
+odoo.define('website_snippet_preset', function (require) {
+    'use strict';
     var snippet_editor = require('web_editor.snippet.editor');
     var core = require('web.core');
     var Dialog = require('web.Dialog');
     var qweb = core.qweb;
     var _t = core._t;
 
+    var OverwriteDialog = Dialog.extend({
```

Comment:
Please respect the order in which things were declared.

Reply:
In this case you get a warning. Use before declaration.

Reply:
Ah ok